### PR TITLE
Fit 355 smoke test

### PR DESCRIPTION
**Pull request by AVandana-OSI**
https://osi-cwds.atlassian.net/browse/FIT-355

**_Description_**: Added a few more test under the smoke test to validate the screening link and snapshot link are working.

**_Command_**: FEATURE_SET=smoke APP_URL=https://web.preint.cwds.io APP_URL_PREFIX=/intake rake

### CODE REVIEW

File: spec/smoke/intake_snapshot_screening_spec.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  end
+end    
+
+describe 'Snapshot Page Ready', type: :feature do
+  before do
+    login_user(path: intake_path)
+  end
+
+  it 'when clicking Start Snapshot link' do
+    Capybara.click_button('Start Snapshot')
```
Comment (wmitchell-cambria) [inline spec/smoke/intake_snapshot_screening_spec.rb]: Should there be an expectation afterwards?

Comment (AVandana-OSI) [reply spec/smoke/intake_snapshot_screening_spec.rb]: @wmitchell-cambria If the Click.button fails the exception is readable.

```
Failures:

  1) Snapshot Page Ready when clicking Start Snapshot link
     Failure/Error: Capybara.click_button('Start Snapshots')

     Capybara::ElementNotFound:
       Unable to find visible button "Start Snapshots" 
```

You want me to add another layer to handle the exception.

Comment (wmitchell-cambria) [reply spec/smoke/intake_snapshot_screening_spec.rb]: Is that `s` intended? Should be `Start Snapshot`, not `Start Snapshots`.

Comment (AVandana-OSI) [reply spec/smoke/intake_snapshot_screening_spec.rb]: got it, added

File: spec/smoke/intake_snapshot_screening_spec.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+describe 'Screening Page Ready', type: :feature do
+  before do
+    login_user(path: intake_path)
+  end
+
+  it 'when clicking Start Screening link' do
+    Capybara.click_button('Start Screening')
```
Comment (edgibbs) [inline spec/smoke/intake_snapshot_screening_spec.rb]: Can delete Capybara. part

**Pull request merged**